src/macros/mod.rs

```diff
@@ -0,0 +1,6 @@
+#[macro_export]
+macro_rules! log {
+    ( $( $t:tt )* ) => {
+        web_sys::console::log_1(&format!( $( $t )* ).into());
```

Comment:
does this `console.log` in the wasm target?

Reply:
yeah, added it for debugging purposes